apps/cowswap-frontend/src/modules/application/containers/App/index.tsx

```diff
@@ -1,63 +1,269 @@
-import { CHRISTMAS_THEME_ENABLED } from '@cowprotocol/common-const'
+import { useMemo, lazy, Suspense } from 'react'
 import { isInjectedWidget } from '@cowprotocol/common-utils'
-
 import ErrorBoundary from 'legacy/components/ErrorBoundary'
-import Footer from 'legacy/components/Footer'
-import Header from 'legacy/components/Header'
 import { URLWarning } from 'legacy/components/Header/URLWarning'
 import TopLevelModals from 'legacy/components/TopLevelModals'
 import DarkModeQueryParamReader from 'legacy/theme'
-
 import { OrdersPanel } from 'modules/account'
 import { useInitializeUtm } from 'modules/utm'
-import { WinterFooter } from 'modules/winterEdition'
-
 import { InvalidLocalTimeWarning } from 'common/containers/InvalidLocalTimeWarning'
 import { useAnalyticsReporter } from 'common/hooks/useAnalyticsReporter'
 import RedirectAnySwapAffectedUsers from 'pages/error/AnySwapAffectedUsers/RedirectAnySwapAffectedUsers'
-
-import { RoutesApp } from './RoutesApp'
 import * as styledEl from './styled'
+import {
+  Color,
+  Media,
+  MenuBar,
+  MenuItem,
+  Footer,
+  ProductVariant,
+  GlobalCoWDAOStyles,
+  LoadingApp,
+} from '@cowprotocol/ui'
+import { CoWDAOFonts } from 'common/styles/CoWDAOFonts'
+
+import IMG_ICON_BRANDED_DOT_RED from '@cowprotocol/assets/images/icon-branded-dot-red.svg'
+import IMG_ICON_COW_RUNNER from '@cowprotocol/assets/cow-swap/game.gif'
+import IMG_ICON_COW_SLICER from '@cowprotocol/assets/cow-swap/ninja-cow.png'
+
+import { useDarkModeManager } from 'legacy/state/user/hooks'
+import { useMediaQuery } from '@cowprotocol/common-hooks'
+
+import { useInjectedWidgetParams } from 'modules/injectedWidget'
+import { useCategorizeRecentActivity } from 'common/hooks/useCategorizeRecentActivity'
+
+import { HeaderControls, HeaderElement } from 'legacy/components/Header/styled'
+import { AccountElement } from 'legacy/components/Header/AccountElement'
+
+import { NetworkSelector } from 'legacy/components/Header/NetworkSelector'
+
+import AppziButton from 'legacy/components/AppziButton'
+import { FortuneWidget } from 'modules/fortune/containers/FortuneWidget'
+import { FeatureGuard } from 'common/containers/FeatureGuard'
+
+const RoutesApp = lazy(() => import('./RoutesApp').then((module) => ({ default: module.RoutesApp })))
```

Comment:
It shouldn't be lazy-loaded

Reply:
Ah, the reason I did it here is so that we'd show a loading animation before the app is fully ready. Please feel free to iterate on this if you think we should fix this.